data/FilterList.json

```diff
@@ -16827,8 +16827,32 @@
   {
     "id": 1471,
     "homeUrl": "http://getadhell.com",
-    "name": "AdHell Standart",
+    "name": "AdHell list",
```

Comment:
Oh, you can keep "Standard" in the title. Just requesting to swap the "t" for a "d" so the spelling is correct (matches the `viewUrl` of standar**d**-package.

Reply:
fuck// okey wait

Reply:
I learn English, and spelling is still a complicated thing.

Reply:
No worries at all. Thanks for all your help.